models/OBAReport.m

```diff
@@ -0,0 +1,38 @@
+//
+//  OBAReport.m
+//  org.onebusaway.iphone
+//
+//  Created by Pho Diep on 3/21/15.
+//  Copyright (c) 2015 OneBusAway. All rights reserved.
+//
+
+#import "OBAReport.h"
+
+@interface NSObject ()
+
+@property (strong, nonatomic) NSDate *timestamp;
```

Comment:
add `readwrite` flag alongside strong and nonatomic for both of these properties.